src/compose.js

```diff
@@ -1,33 +1,26 @@
-var _compose = require('./internal/_compose');
-var _createComposer = require('./internal/_createComposer');
+var pipe = require('./pipe');
+var reverse = require('./reverse');
 
 
 /**
- * Creates a new function that runs each of the functions supplied as parameters in turn,
- * passing the return value of each function invocation to the next function invocation,
- * beginning with whatever arguments were passed to the initial invocation.
- *
- * Note that `compose` is a right-associative function, which means the functions provided
- * will be invoked in order from right to left. In the example `var h = compose(f, g)`,
- * the function `h` is equivalent to `f( g(x) )`, where `x` represents the arguments
- * originally passed to `h`.
+ * Performs right-to-left function composition. The rightmost function may have
+ * any arity; the remaining functions must be unary.
  *
  * @func
  * @memberOf R
  * @category Function
- * @sig ((y -> z), (x -> y), ..., (b -> c), (a... -> b)) -> (a... -> z)
- * @param {...Function} functions A variable number of functions.
- * @return {Function} A new function which represents the result of calling each of the
- *         input `functions`, passing the result of each function call to the next, from
- *         right to left.
+ * @sig ((y -> z), (x -> y), ..., (a -> b)) -> (a -> b -> c -> ... -> y -> z)
+ * @param {...Function} functions
+ * @return {Function}
  * @example
  *
- *      var triple = function(x) { return x * 3; };
- *      var double = function(x) { return x * 2; };
- *      var square = function(x) { return x * x; };
- *      var squareThenDoubleThenTriple = R.compose(triple, double, square);
+ *      var f = R.compose(R.dec, Math.pow);
  *
- *      //≅ triple(double(square(5)))
- *      squareThenDoubleThenTriple(5); //=> 150
+ *      f(3, 4); // 3^4 - 1
```

Comment:
This example is an improvement, but I'd like to capture that it can take more than two functions in the example.  Perhaps we could add back `double` and incorporate that?  `R.compose(R.dec, double, Math.pow)` ?